cognite/neat/core/_utils/spreadsheet.py

```diff
@@ -98,6 +98,10 @@ def read_individual_sheet(
         # Special handling for Value Type column, #N/A is treated specially by NEAT it means Unknown
         raw["Value Type"] = raw["Value Type"].replace(float("nan"), "#N/A")
 
+    if "Concept" in raw.columns:
```

Comment:
"Concept" is always hardcoded like this or is it possible to have variations (small caps, all caps etc etc)?